composer.json

```diff
@@ -28,7 +28,8 @@
         "ext-openssl": "*",
         "ext-libxml": "*",
         "ext-simplexml": "*",
-        "ext-json": "*"
+        "ext-json": "*",
+        "ext-intl": "*"
```

Comment:
Place this in the `"suggest"` section instead.

Reply:
This is needed for `IntlDateFormatter`, which, in turn, is needed to parse the localized date formats. For any platform that supports multiple languages, this handles all their local nuances automatically, so I'd argue that this is a valuable addition and sooner or later other Bridge will need it anyway.

Reply:
Ping @dvikan